requirements.txt

```diff
@@ -1,7 +1,7 @@
 black==22.3.0
 flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@v0.5.0
+graviton@git+https://github.com/algorand/graviton@91fe1f6f82b2a75601cf28e94b8bb2b5c7186621
```

Comment:
```suggestion
graviton@git+https://github.com/algorand/graviton@v0.6.0
```

Reply:
Tests are failing on Graviton's main branch. This PR fixes the problem: https://github.com/algorand/graviton/pull/41

Once that is merged, I'll create tag `v0.6.0` and the tests here should pass as well.

Reply:
Thanks for pointing out, I will take a pass on that one soon.